src/server/api/controllers/eventsNews.js

```diff
@@ -0,0 +1,184 @@
+  featureImageFilename
+`;
+
+const dateTimeFields = `
+  iso
+  date
+  month
+  monthSym
+  year
+`;
+
+const dateTimeFieldsEvents = `
+  startDateTime {
+    ${dateTimeFields}
+  }
+  endDateTime {
+    ${dateTimeFields}
+  }
+`;
+
+const dateTimeFieldsNews = `
+  datetime {
+    ${dateTimeFields}
+  }
+`;
+
+const fullEventsQuery = `
+  ${basicFields}
+  ${dateTimeFieldsEvents}
+`;
+
+const fullNewsQuery = `
+  ${basicFields}
+  ${dateTimeFieldsNews}
+`;
+
+const sortNews = (list) =>
+  list.sort((a, b) =>
+    new Date(b.datetime.iso) - new Date(a.datetime.iso));
+
+const sortEvents = (list) =>
+  list.sort((a, b) =>
+    new Date(b.startDateTime.iso) - new Date(a.startDateTime.iso));
+
+export function getNewsItem(req, res) {
+  const body = `
+    query {
+      news(id: "${req.params.id}") {
+        ${fullNewsQuery}
+      }
+    }
+  `;
+
+  fetch(badgerBrainEndpoint, getRequestOptions(req, body))
+    .then((response) => {
+      response.json().then((json) => {
+        res.send(json.data ? json.data.event : json);
+      });
+    })
+    .catch((err) => {
+      res.status(err.status).send(err.message);
+    });
+}
+
+export function getNews(req, res) {
+  const body = `
+    query {
+      allNews {
+        ${fullNewsQuery}
+      }
+    }
+  `;
+
+  fetch(badgerBrainEndpoint, getRequestOptions(req, body))
+    .then((response) => response.json())
+    .then((news) =>
+      res.send({ list: sortNews(news.data.allNews) })
+    .catch((err) => {
+      res.status(err.status).send(err.message);
+    }));
+}
+
+export function getEvent(req, res) {
+  const body = `
+    query {
+      event(id: "${req.params.id}") {
+        ${fullEventsQuery}
+      }
+    }
+  `;
+
+  fetch(badgerBrainEndpoint, getRequestOptions(req, body))
+    .then((response) => {
+      response.json().then((json) => {
+        res.send(json.data ? json.data.event : json);
+      });
+    })
+    .catch((err) => {
+      res.status(err.status).send(err.message);
+    });
+}
+
+export function getEvents(req, res) {
+  const body = `
+    query {
+      allEvents {
+        ${fullEventsQuery}
+      }
+    }
+  `;
+
+  fetch(badgerBrainEndpoint, getRequestOptions(req, body))
+    .then((response) => response.json())
+    .then((events) => {
+      res.send({ list: sortEvents(events.data.allEvents) });
+    })
+    .catch((err) => {
+      res.status(err.status).send(err.message);
+    });
+}
+
+export function getTags(req, res) {
+  const body = `
+    query {
+      allEvents {
+        ${fullEventsQuery}
+      }
+      allNews {
+        ${fullNewsQuery}
+      }
+    }
+  `;
+
+  fetch(badgerBrainEndpoint, getRequestOptions(req, body))
+    .then((response) => response.json())
+    .then((tags) => {
+      res.send({
+        events: {
+          list: sortEvents(tags.data.allEvents),
+        },
+        news: {
+          list: sortNews(tags.data.allNews),
+        },
+      });
+    })
+    .catch((err) => {
+      res.status(err.status).send(err.message);
```

Comment:
Also I think they're all the same. We could extract them into a single function :)